module-2/index.ts

```diff
@@ -0,0 +1,8 @@
+import { app } from './app';
+import { AddressInfo } from 'net';
+
+const PORT = 4000;
+const server = app.listen(PORT, '0.0.0.0', () => {
```

Comment:
This part is an overkill for the project level you're building :)
A simple `app.listen(PORT, () => { console.log(`...`); });` would have been enough.

Reply:
Simplified 😀